.github/workflows/docs.yml

```diff
@@ -64,6 +64,7 @@ jobs:
         run: |
           poetry run mkdocs build --verbose --strict --config-file mkdocs.insiders.yml
           aws s3 sync ./site "s3://docs.kolena.io"
+          aws lambda invoke --function-name docs-cdn-invalidation-lambda --invocation-type RequestResponse --no-cli-pager --region us-west-2 /dev/stdout
```

Comment:
Have not tried, but would this also work (or any concern) ?

```
aws cloudfront create-invalidation --distribution-id {{ secret.distribution_id }} --paths "/*"
```

Reply:
This should work, let me check. Original thinking was use lambda to only invalidate the file that changed, but that's very hard to do. We can cut the middle man